python/paddle/v2/fluid/initializer.py

```diff
@@ -302,7 +312,7 @@ class MSRAInitializer(Initializer):
             (https://arxiv.org/abs/1502.01852)
     """
 
-    def __init__(self, uniform=True, fan_in=None, seed=0):
+    def __init__(self, uniform=True, fan_in=None, seed=set_random_seed(0)):
```

Comment:
Could the user set the seed globally instead of setting the seed at each parameter initializer?

Reply:
I didn't get an elegant way to implement it.  Use a global seed seems ugly,

Reply:
fixed